recipes/googleapis/all/conanfile.py

```diff
@@ -119,40 +132,47 @@ def deactivate_library(key):
         #  - Inconvenient macro names from usr/include/sys/syslimits.h in some macOS SDKs: GID_MAX
         #    Patched here: https://github.com/protocolbuffers/protobuf/commit/f138d5de2535eb7dd7c8d0ad5eb16d128ab221fd
         #    as of 3.21.4 issue still exist
-        if tools.Version(self.deps_cpp_info["protobuf"].version) <= "3.21.5" and self.settings.os == "Macos":
+        if Version(self.deps_cpp_info["protobuf"].version) <= "3.21.5" and self.settings.os == "Macos":
```

Comment:
should be replace by `self.dependencies["protobuf"].ref.version` I guess, but according to https://docs.conan.io/en/latest/reference/conanfile/dependencies.html#dependencies-interface, it should be used in `validate()` and `generate()` only, and here we are in `build()`.